src/App.tsx

```diff
@@ -91,6 +92,8 @@ function App() {
                 <hr></hr>
                 <h5>Assigned to me</h5>
                 <RevealTicket></RevealTicket>
+                <hr></hr>
+                <EditTicket ticket={ticketDB.getCentralList()[1]}></EditTicket>
```

Comment:
I think this is getting stored in App.tsx

Reply:
What do you mean?

Reply:
ticketDB seems to not be used right now, we have the lists in App.tsx on a different branch, you might have issues when you merge.

Reply:
feature/fix-lists implements a replacement for the ticketdb. Once feature/fix-lists gets merged I think we could replace this with something like ticket={centralList} and centralList will be passed from the App.tsx file.

Reply:
This is normally resolved now.

Reply:
Normally, the change is made by now.